en/matrix.md

```diff
@@ -3,8 +3,10 @@ layout: page
 title: Bitcoin Feature Matrix
 permalink: /en/matrix/
 nowrap: true
+redirect_from:
+  - /en/compatibility/
+  - en/compatibility/wasabi/
```

Comment:
Might need to be:
```suggestion
  - /en/compatibility/wasabi/
```

Reply:
I updated this locally and pushed to 2024-09-matrix